src/main/python/ttconv/tt.py

```diff
@@ -225,6 +270,11 @@ def validate(args):
   print(args.input)
   LOGGER.info("Input file is %s", args.input)
 
+# Ensure that the handler is added only once/globally
+# Otherwise the handler will be called multiple times
+progress = ProgressConsoleHandler()
```

Comment:
No need to define the `progress` module variable

Reply:
fixed